pkgs/development/web/nodejs/v22.nix

```diff
@@ -15,9 +17,11 @@ buildNodejs {
     ./bypass-darwin-xcrun-node16.patch
     ./node-npm-build-npm-package-logic.patch
     ./use-correct-env-in-tests.patch
+    ./bin-sh-node-run-v22.patch
     (fetchpatch2 {
-      url = "https://github.com/nodejs/node/commit/534c122de166cb6464b489f3e6a9a544ceb1c913.patch";
-      hash = "sha256-4q4LFsq4yU1xRwNsM1sJoNVphJCnxaVe2IyL6AeHJ/I=";
+      url = "https://github.com/nodejs/node/pull/53373.patch";
```

Comment:
```suggestion
      url = "https://github.com/nodejs/node/commit/14863e80584e579fd48c55f6373878c821c7ff7e.patch";
```

Reply:
Why use commit URL instead of PR patch URL?

Reply:
The PR might get deleted, the commit not. Also, we should pick the commit that landed on nodejs/node, not the commits that led to that commit.

Reply:
I don’t think it’s possible to delete PR without deleting repository (and the commit can be GCed if it is unreachable), but I understand your point w.r.t using landed commits. I’ll update this (and other PR URLs) tomorrow to use commit patch URLs.